### PR TITLE
upgrade for 0.11

### PR DESCRIPTION
**Pull request by justinwoo**

### CODE REVIEW

**Comment by justinwoo** [comment]
updated node target version to 6 since 5 doesn't run the install script and the new LTS is 6 anyhow.

File: src/Pux.purs [hunk 1]
```diff
@@ -59,11 +59,11 @@ start config = do
       htmlSignal = stateSignal ~> config.view
       mapAffect affect = launchAff $ unsafeCoerceAff do
         ev <- affect
-        later $ case ev of
+        case ev of
```
Comment (sloosch) [inline src/Pux.purs]: ````later```` was replaced with ````delay```` (https://github.com/slamdata/purescript-aff/commit/1ed0d751dd7427074f98d86220566a58572d5aa7)
```delay (Milliseconds 0.0)``` should then be placed above the case

File: src/Pux.purs [hunk 2]
```diff
@@ -59,11 +59,11 @@ start config = do
...
           Nothing -> pure unit
           Just e -> liftEff $ send evChannel (singleton e)
       effectsSignal = effModelSignal ~> map mapAffect <<< _.effects
-  runSignal $ effectsSignal ~> sequence_
+  _ <- runSignal $ effectsSignal ~> sequence_
```
Comment (sloosch) [inline src/Pux.purs]: you don`t need to discard `Unit` explicitly

File: src/Pux/Router.purs [hunk 3]
```diff
@@ -151,7 +151,7 @@ parseQuery s = Query <<< M.fromFoldable <<< catMaybes <<< map part2tuple $ parts
   part2tuple :: String -> Maybe (Tuple String String)
   part2tuple part = do
     let param' = S.split (S.Pattern "=") part
-    guard $ A.length param' == 2
+    _ <- guard $ A.length param' == 2
```
Comment (sloosch) [inline src/Pux/Router.purs]: you don't need to discard ``Unit`` explicitly. import Prelude.discard

File: src/Pux.purs [hunk 4]
```diff
@@ -59,11 +59,11 @@ start config = do
       htmlSignal = stateSignal ~> config.view
       mapAffect affect = launchAff $ unsafeCoerceAff do
         ev <- affect
-        later $ case ev of
```
Comment (sloosch) [inline src/Pux.purs:62]: ````later```` was replaced with ````delay```` (https://github.com/slamdata/purescript-aff/commit/1ed0d751dd7427074f98d86220566a58572d5aa7)
```delay (Milliseconds 0.0)``` should therefore be placed above the `case`

File: src/Pux.purs (hunk 2 above)
Comment (sloosch) [inline src/Pux.purs]: you don't need to discard ``Unit`` explicitly. import Prelude.discard

**Review by sloosch — CHANGES_REQUESTED** [review]

File: src/Pux.purs (hunk 4 above)
Comment (justinwoo) [reply src/Pux.purs:62]: seems like a smell, but yeah, updated

**Comment by justinwoo** [comment]
updated with @sloosch's suggestions

**Pull request closed without merging**